src/CapAndHomalg.jl

```diff
@@ -67,21 +67,21 @@ include(joinpath("..", "deps", "homalg-project.jl"))
 import Pkg
 import Markdown
 
-Base.:*(x::GapObj, y::String) = x * GapObj(y)
-Base.getindex(x::GapObj, y::String) = GAP.Globals.ELM_LIST(x, GapObj(y))
-Base.getindex(x::GapObj, y::GapObj) = GAP.Globals.ELM_LIST(x, y)
-Base.:/(x::Any, y::GapObj) = GAP.Globals.QUO(GapObj(x, recursive=true), y)
-Base.:/(x::GapObj, y::Array{GapObj,1}) = GAP.Globals.QUO(x, GapObj(y))
-Base.:/(x::GapObj, y::Array) = GAP.Globals.QUO(x, GapObj(y, recursive=true))
-
-function Base.showable(mime::MIME, obj::GapObj)
+CapAndHomalg.:*(x::GapObj, y::String) = x * GapObj(y)
```

Comment:
But `CapAndHomalg.:*` is just an alias for `Base.:*`, so this PR really does nothing.

Can also be verified in the REPL:
```
julia> import CapAndHomalg
CapAndHomalg vnot installed
Imported OSCAR's components GAP and Singular_jll
Type: ?CapAndHomalg for more information

julia> CapAndHomalg.:*
* (generic function with 333 methods)

julia> CapAndHomalg.:* === Base.:*
true

julia> x = CapAndHomalg.GAP.GapObj(BigInt(2)^100)
GAP: 1267650600228229401496703205376

julia> y = CapAndHomalg.GAP.GapObj([x])
GAP: [ 1267650600228229401496703205376 ]

julia> x / [y]  # this would error with just GAP.jl loaded
GAP: [ [ 1 ] ]
```